lib/entities/default_settings_migration.dart

```diff
@@ -766,6 +769,19 @@ Future<void> changeDefaultMoneroNode(
   }
 }
 
+Future<void> fixBtcDerivationPaths(Box<WalletInfo> walletsInfoSource) async {
+  for (WalletInfo walletInfo in walletsInfoSource.values) {
+    if (walletInfo.type == WalletType.bitcoin ||
+        walletInfo.type == WalletType.bitcoinCash ||
+        walletInfo.type == WalletType.litecoin) {
+      if (walletInfo.derivationInfo?.derivationPath == "m/0'/0") {
+        walletInfo.derivationInfo!.derivationPath = "m/0'";
+      }
+    }
+    await walletInfo.save();
```

Comment:
**Enhancement**
no need to call save for every wallet, while we only need to call it if we change the derivation path
this code is executed on app start, so it might slow the start of the app at that one time for no reason